<ui version="4.0" >
 <class>ProjectProperty</class>
 <widget class="QDialog" name="ProjectProperty" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Project property</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabProjectDefinition" >
      <attribute name="title" >
       <string>&amp;Project Definition</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>9</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>9</number>
       </property>
       <property name="bottomMargin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Project</string>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>9</number>
          </property>
          <property name="topMargin" >
           <number>9</number>
          </property>
          <property name="rightMargin" >
           <number>9</number>
          </property>
          <property name="bottomMargin" >
           <number>9</number>
          </property>
          <property name="horizontalSpacing" >
           <number>6</number>
          </property>
          <property name="verticalSpacing" >
           <number>6</number>
          </property>
          <item row="2" column="2" >
           <widget class="QToolButton" name="m_logButton" >
            <property name="icon" >
             <iconset resource="../application.qrc" >:/images/folder.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_12" >
            <property name="text" >
             <string>Project &amp;Log :</string>
            </property>
            <property name="buddy" >
             <cstring>m_logLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Project &amp;Directory :</string>
            </property>
            <property name="buddy" >
             <cstring>m_projectLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="label_9" >
            <property name="text" >
             <string>Project &amp;RCS :</string>
            </property>
            <property name="buddy" >
             <cstring>m_projectRCSComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Project &amp;Name :</string>
            </property>
            <property name="buddy" >
             <cstring>m_nameLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="m_nameLineEdit" >
            <property name="whatsThis" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This is the project name showed in the title bar of &lt;span style=" font-weight:600;">XINX&lt;/span>.This just an informative information useless.&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QToolButton" name="m_projectButton" >
            <property name="icon" >
             <iconset resource="../application.qrc" >:/images/folder.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="DirectoryEdit" name="m_projectLineEdit" >
            <property name="whatsThis" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This is the project directory (generally where is stored &lt;span style=" font-style:italic;">configuration.xml&lt;/span>). This is the root of the project directory dock, used to search file and for &lt;span style=" font-style:italic;">revision control system&lt;/span> access.&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QComboBox" name="m_projectRCSComboBox" >
            <property name="whatsThis" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This the list of &lt;span style=" font-style:italic;">revision control system&lt;/span>. The list is appended by modules that you can find in the customize dialog.&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="DirectoryEdit" name="m_logLineEdit" >
            <property name="whatsThis" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This is the path of the log server directory (where is stored all &lt;span style=" font-style:italic;">presentation_*.xml&lt;/span> files).&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDefaultValue" >
      <attribute name="title" >
       <string>&amp;Default value</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>9</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>9</number>
       </property>
       <property name="bottomMargin" >
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Default Value</string>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>9</number>
          </property>
          <property name="topMargin" >
           <number>9</number>
          </property>
          <property name="rightMargin" >
           <number>9</number>
          </property>
          <property name="bottomMargin" >
           <number>9</number>
          </property>
          <property name="horizontalSpacing" >
           <number>6</number>
          </property>
          <property name="verticalSpacing" >
           <number>6</number>
          </property>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_8" >
            <property name="text" >
             <string>Version :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2" >
           <widget class="QComboBox" name="m_langComboBox" >
            <property name="whatsThis" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This is the default lang used by your project. This can be used for the search path of the &lt;span style=" text-decoration: underline;">C&lt;/span>ustom files tab.&lt;/p>&lt;/body>&lt;/html></string>
            </property>
            <item>
             <property name="text" >
              <string>FRA</string>
             </property>
             <property name="icon" >
              <iconset resource="../application.qrc" >:/images/france.png</iconset>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>ENG</string>
             </property>
             <property name="icon" >
              <iconset resource="../application.qrc" >:/images/usa.png</iconset>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>DEU</string>
             </property>
             <property name="icon" >
              <iconset resource="../application.qrc" >:/images/germany.png</iconset>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>ITA</string>
             </property>
             <property name="icon" >
              <iconset resource="../application.qrc" >:/images/italy.png</iconset>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>ESP</string>
             </property>
             <property name="icon" >
              <iconset resource="../application.qrc" >:/images/spain.png</iconset>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>POR</string>
             </property>
             <property name="icon" >
              <iconset resource="../application.qrc" >:/images/portugal.png</iconset>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>N&amp;avigator</string>
            </property>
            <property name="buddy" >
             <cstring>m_navigatorComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2" >
           <widget class="QComboBox" name="m_navigatorComboBox" >
            <property name="whatsThis" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This is the default navigator used in the list of path in the &lt;span style=" text-decoration: underline;">C&lt;/span>ustom files tab.&lt;/p>&lt;/body>&lt;/html></string>
            </property>
            <item>
             <property name="text" >
              <string>NAV/MIC</string>
             </property>
             <property name="icon" >
              <iconset resource="../application.qrc" >:/images/ie_logo.png</iconset>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>NAV/NET</string>
             </property>
             <property name="icon" >
              <iconset resource="../application.qrc" >:/images/firefox_logo.png</iconset>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1" colspan="2" >
           <widget class="QLabel" name="m_configurationVersionLabel" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>&amp;Lang</string>
            </property>
            <property name="buddy" >
             <cstring>m_langComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="label_13" >
            <property name="text" >
             <string>Presentation file :</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2" >
           <widget class="QLabel" name="m_presentationLabel" >
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabWebServices" >
      <attribute name="title" >
       <string>&amp;WebServices</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>9</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>9</number>
       </property>
       <property name="bottomMargin" >
        <number>9</number>
       </property>
       <property name="horizontalSpacing" >
        <number>6</number>
       </property>
       <property name="verticalSpacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QGroupBox" name="m_webServiceGroupBox" >
         <property name="whatsThis" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Check this box is you want add service web to the project. The list is a list of WSDL. When you start the project, &lt;span style=" font-weight:600;">XINX&lt;/span> will connect to the service and download the description file.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Next you can execute a service stream that open a dialog with the in stream and the out stream.&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="title" >
          <string>Project has WebServices</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="XinxListWidgetImpl" native="1" name="m_servicesList" />
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCustomFiles" >
      <attribute name="title" >
       <string>&amp;Custom files</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="m_specifiqueGroupBox" >
         <property name="whatsThis" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">If checked, &lt;span style=" font-weight:600;">XINX&lt;/span> will check if the file is standard or not if the file is standard, and in this case the project porpose you to prefix the file.&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="title" >
          <string>&amp;Project is derivated from standard</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="m_pathListLabel" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>&amp;Search path list :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2" >
           <widget class="XinxListWidgetImpl" native="1" name="m_searchPathList" />
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="m_prefixLabel" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Project p&amp;refix :</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2" >
           <widget class="XinxListWidgetImpl" native="1" name="m_prefixList" />
          </item>
          <item row="4" column="0" >
           <widget class="QLabel" name="label_10" >
            <property name="text" >
             <string>&amp;Specifique project path name :</string>
            </property>
            <property name="buddy" >
             <cstring>m_specifiqueProjectPathLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1" >
           <widget class="QLineEdit" name="m_specifiqueProjectPathLineEdit" >
            <property name="whatsThis" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">The sub-directory where the derivated file must be stored.&lt;/p>&lt;/body>&lt;/html></string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="icon" >
        <iconset resource="../application.qrc" >:/images/button_ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="../application.qrc" >:/images/button_cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DirectoryEdit</class>
   <extends>QLineEdit</extends>
   <header>directoryedit.h</header>
  </customwidget>
  <customwidget>
   <class>XinxListWidgetImpl</class>
   <extends>QWidget</extends>
   <header>xinxlistwidgetimpl.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_nameLineEdit</tabstop>
  <tabstop>m_projectLineEdit</tabstop>
  <tabstop>m_projectButton</tabstop>
  <tabstop>m_langComboBox</tabstop>
  <tabstop>m_navigatorComboBox</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="../application.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ProjectProperty</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>466</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ProjectProperty</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>547</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
